frontend/server/src/Controllers/Contest.php

```diff
@@ -5,10 +5,12 @@
 /**
  * ContestController
  *
+ * @psalm-type Statement=array{markdown: string, statementType: string, gitObjectId?: string}
```

Comment:
para que no se confunda con una redacción.

```suggestion
 * @psalm-type PrivacyStatement=array{markdown: string, statementType: string, gitObjectId?: string}
```

Reply:
Listo